overview/templates/overview/candidate_detail.html

```diff
@@ -336,6 +336,92 @@ <h2 id="questionnaires">Questionnaires</h2>
     </div>
 </div>
 
+{% if user.is_staff %}
+<div class="row">
+    <div class="col-md-6">
+        <h2 id="">Quotes</h2>
+            {% for quote in candidate.quote_set.all %}
+                {% if quote.display %}
+                    <blockquote> 
+                        <p>{{ quote.quote|markdown }}</p>
+                            <footer>
+                                {% if quote.by %} {{ quote.by }} {% else %} {{ candidate.fullname }}{% endif %}{% if quote.cite %}, <cite>{{ quote.cite }}</cite>{% endif %}
+                            </footer>
+                        </blockquote>
+                {% endif %}
+            {% endfor %}
+    </div>
+    <div class="col-md-6">
+        <h2 id="">Press</h2>
+        {% if candidate.pressarticlecandidate_set.all %}
+            <ul>
+            {% for article in candidate.pressarticlecandidate_set.all %}
+                {% if article.display and not article.candidate_is_the_author %}
+                    <li><p>
+                        {% if article.pressarticle.author %}{{article.pressarticle.author}}.{% endif %}
+                        {% if article.pressarticle.link %}
+                            <a href="{{ article.pressarticle.link }}" target="_blank">
+                            {{ article.pressarticle.title }}.
+                            <span class="fa fa-external-link">
+                        {% else %}
+                            {{ article.pressarticle.title }}.
+                        {% endif %}
+                        </a>
+                        <br>
+                        <i>{{ article.pressarticle.pressoutlet.name }}</i> {% if article.pressarticle.date %}{{ article.pressarticle.date }}{% endif %}.
+                        {% if article.sample %}
+                            <a href="#!" 
+                               data-toggle="popover" 
+                               data-trigger="focus"
+                               data-html="true" 
+                               data-container="body" 
+                               data-content='{{ article.sample }}'>
```

Comment:
this might choke if the article has something like `Jan's second time running`

Reply:
quotes suck